### PR TITLE
Renamed 'klass_id' columns to 'klass_key'

### PR DESCRIPTION
**Pull request by gsidebo**
#### What are the relevant tickets?
No ticket. This is based on some weird behavior we observed due to the naming of a model field/db column

#### What's this PR do?
Changes the names of `Klass.klass_id` and `Line.klass_id` to `Klass.klass_key` and `Line.klass_key`

#### How should this be manually tested?
Migrate your database, click through the app, make sure everything looks normal

### CODE REVIEW

**Comment by gsidebo** [comment]
@giocalitri Fixed those variable names (I think)

File: klasses/api.py [hunk 1]
```diff
@@ -27,12 +27,12 @@ def serialize_user_klasses(user):
...
     # extract the payments already done.
-    klass_ids_in_lines = list(Line.fulfilled_for_user(user).values_list(
-        'klass_id', flat=True).distinct())
+    klass_keys_in_lines = list(Line.fulfilled_for_user(user).values_list(
+        'klass_key', flat=True).distinct())
 
     bootcamp_client = BootcampAdmissionClient(user.email)
-    all_klasses_ids = list(set(klass_ids_in_lines).union(set(bootcamp_client.payable_klasses_ids)))
-    klasses_qset = Klass.objects.filter(klass_id__in=all_klasses_ids).order_by('klass_id')
+    all_klasses_ids = list(set(klass_keys_in_lines).union(set(bootcamp_client.payable_klasses_ids)))
```
Comment (giocalitri) [inline klasses/api.py]: sorry, I wrote this before your second commit

File: klasses/bootcamp_admissions_client.py [hunk 2]
```diff
@@ -37,17 +37,17 @@ def _get_admissions(self):
             "user":"foo@example.com",
             "bootcamps":[
                 {
-                    "bootcamp_id":6,
+                    "bootcampgi_id":6,
```
Comment (gsidebo) [inline klasses/bootcamp_admissions_client.py]: fingers falling asleep on the keyboard 🖐 😴

**Comment by gsidebo** [comment]
@giocalitri looks like i was limiting my find/replace to *.py files. i think all comments have been addressed now

**Comment by giocalitri** [comment]
question before I run the code in my local instance: does the migration preserve the data?

**Comment by giocalitri** [comment]
I think this is good to go 👍

**Pull request merged**